src/common/modules/AutocorrectHandler.js

```diff
@@ -291,3 +272,22 @@ BrowserCommunication.addListener(COMMUNICATION_MESSAGE_TYPE.AUTOCORRECT_BACKGROU
 
     return sendSettings(request.optionValue);
 });
+
+browser.runtime.onMessage.addListener((message, sender) => {
+    // console.log(message);
```

Comment:
dead debug code

Reply:
This was already there, but will remove.